composer.json

```diff
@@ -22,39 +24,28 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": "^7.3",
         "ext-pdo": "*",
-        "ezsystems/ezpublish-kernel": ">=5.4|>=2014.11",
+        "ezsystems/ezpublish-kernel": "^8.0@dev",
```

Comment:
same here: we have to append the requirements for 8.0 to the existing 5.4|2014.11 requirements (unless this is meant to be a non-compatible branch...)